app/components/nav-bar.js

```diff
@@ -9,6 +9,17 @@ export default class NavBar extends Component {
     return !(String(this.session.currentRouteName).includes('public'));
   }
 
+    @computed('session.currentRouteName')
+  get isNotWizardPage() {
+    const wizardPages = String(this.session.currentRouteName).includes('events.view.edit.basic-details')
+    || String(this.session.currentRouteName).includes('events.view.edit.attendee')
+    || String(this.session.currentRouteName).includes('events.view.edit.sessions-speakers')
+    || String(this.session.currentRouteName).includes('events.view.edit.index')
+    || String(this.session.currentRouteName).includes('events.view.edit.sponsors')
+    || String(this.session.currentRouteName).includes('events.view.edit.other-details');
+    return !(wizardPages);
```

Comment:
you can do everything in a single line.
 return String(this.session.currentRouteName).includes('events.view.edit')

Reply:
also you should do the same thing for create page
return String(this.session.currentRouteName).includes('create')